corehq/apps/registration/utils.py

```diff
@@ -282,3 +286,20 @@ def create_30_day_trial(domain_obj):
     )
     trial_subscription.is_active = True
     trial_subscription.save()
+
+
+def create_free_community_plan(domain_obj):
+    community_plan_version = DefaultProductPlan.get_default_plan_by_domain(
+        domain_obj, edition=SoftwarePlanEdition.COMMUNITY
+    )
+    community_account = BillingAccount.objects.get_or_create(
+        name="Free Community Account for %s" % domain_obj.name,
```

Comment:
@biyeun or @nickpell  I have tested with this setup and everything works but can you help me with following accounting/billing questions?
1. Is naming the account this way okay?
2. What should [account_type](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/accounting/models.py#L61) be, `USER_CREATED`? 
3. Since this is Free Community plan, I am not setting any `pre_or_post_pay `, is this alright?
4. Since the plan is Free, I didn't set a subcription end date as well.

It wasn't super obvious from the code what these should be set as. Should these questions go to the Spec lead instead?